easybuild/easyconfigs/c/Circuitscape/Circuitscape-5.11.2-Julia-1.7.2.eb

```
@@ -0,0 +1,322 @@
+        'checksums': ['8af4dac29839e796a651b8cb6a325b6f9c7eee5d5620b3e366b64f58325ebeb7'],
+    }),
+    ('FixedPointNumbers', '0.8.4', {
+        'source_urls': ['https://github.com/JuliaMath/FixedPointNumbers.jl/archive/'],
+    }),
+    ('ColorTypes', '0.11.1', {
+        'source_urls': ['https://github.com/JuliaGraphics/ColorTypes.jl/archive/'],
+    }),
+    ('Adapt', '3.4.0', {
+        'source_urls': ['https://github.com/JuliaGPU/Adapt.jl/archive/'],
+    }),
+    ('OffsetArrays', '1.12.8', {
+        'source_urls': ['https://github.com/JuliaArrays/OffsetArrays.jl/archive/'],
+    }),
+    ('DiskArrays', '0.3.7', {
+        'source_urls': ['https://github.com/meggart/DiskArrays.jl/archive/'],
+    }),
+    ('Extents', '0.1.1', {
+        'source_urls': ['https://github.com/rafaqz/Extents.jl/archive/'],
+    }),
+    ('JpegTurbo_jll', '2.1.2+0', {
+        'source_urls': ['https://github.com/JuliaBinaryWrappers/JpegTurbo_jll.jl/archive/'],
+        'sources': ['JpegTurbo-v%(version)s.tar.gz'],
+    }),
+    ('LERC_jll', '3.0.0+1', {
+        'source_urls': ['https://github.com/JuliaBinaryWrappers/LERC_jll.jl/archive/'],
+        'sources': ['LERC-v%(version)s.tar.gz'],
+    }),
+    ('Zstd_jll', '1.5.2+0', {
+        'source_urls': ['https://github.com/JuliaBinaryWrappers/Zstd_jll.jl/archive/'],
+        'sources': ['Zstd-v%(version)s.tar.gz'],
+    }),
+    ('Libtiff_jll', '4.4.0+0', {
+        'source_urls': ['https://github.com/JuliaBinaryWrappers/Libtiff_jll.jl/archive/'],
+        'sources': ['Libtiff-v%(version)s.tar.gz'],
+    }),
+    ('SQLite_jll', '3.40.0+0', {
+        'source_urls': ['https://github.com/JuliaBinaryWrappers/SQLite_jll.jl/archive/'],
+        'sources': ['SQLite-v%(version)s.tar.gz'],
+    }),
+    ('PROJ_jll', '900.100.0+0', {
+        'source_urls': ['https://github.com/JuliaBinaryWrappers/PROJ_jll.jl/archive/'],
+        'sources': ['PROJ-v%(version)s.tar.gz'],
+    }),
+    ('libgeotiff_jll', '100.700.100+0', {
+        'source_urls': ['https://github.com/JuliaBinaryWrappers/libgeotiff_jll.jl/archive/'],
+        'sources': ['libgeotiff-v%(version)s.tar.gz'],
+    }),
+    ('Kerberos_krb5_jll', '1.19.3+0', {
+        'source_urls': ['https://github.com/JuliaBinaryWrappers/Kerberos_krb5_jll.jl/archive/'],
+        'sources': ['Kerberos_krb5-v%(version)s.tar.gz'],
+    }),
+    ('OpenSSL_jll', '1.1.19+0', {
+        'source_urls': ['https://github.com/JuliaBinaryWrappers/OpenSSL_jll.jl/archive/'],
+        'sources': ['OpenSSL-v%(version)s.tar.gz'],
+    }),
+    ('LibPQ_jll', '14.3.0+1', {
+        'source_urls': ['https://github.com/JuliaBinaryWrappers/LibPQ_jll.jl/archive/'],
+        'sources': ['LibPQ-v%(version)s.tar.gz'],
+    }),
+    ('HDF5_jll', '1.12.2+2', {
+        'source_urls': ['https://github.com/JuliaBinaryWrappers/HDF5_jll.jl/archive/'],
+        'sources': ['HDF5-v%(version)s.tar.gz'],
+    }),
+    ('libpng_jll', '1.6.38+0', {
```

It seems that there is something wrong with the packaging of this version of `libpng_jll`, the previous release does work though:
```suggestion
    ('libpng_jll', '1.6.37+5', {
```